components/CreateProfile.js

```diff
@@ -172,6 +166,9 @@ const CreateProfile = ({
   onSubmit,
   onSecondaryAction,
   emailAlreadyExists,
+  isOauth,
```

Comment:
```suggestion
  isOAuth,
```

Reply:
Thanks. I've changed `isOauth` to `isOAuth` at all places. 👍🏽